test/parallel/test-repl.js

```diff
@@ -863,7 +863,18 @@ alternatively use dynamic import: const moduleName = await import(\'module-name\
       '',
       'Uncaught:',
       'SyntaxError: Cannot use import statement inside the Node.js REPL, \
-alternatively use dynamic import: const {export1:localName1,export2} = await import(\'bar\');',
+alternatively use dynamic import: await import(\'bar\');',
+    ]
+  },
+  {
+    send: 'import {default as alias} from "bar";',
+    expect: [
+      kSource,
+      kArrow,
+      '',
+      'Uncaught:',
+      'SyntaxError: Cannot use import statement inside the Node.js REPL, \
+alternatively use dynamic import: await import(\'bar\');',
```

Comment:
```suggestion
alternatively use dynamic import: const alias = await import(\'bar\');',
```

Reply:
Based on [this](https://github.com/nodejs/node/pull/48129#discussion_r1212690038) I thought we don't need the `const` declaration